discovery/service.go

```diff
@@ -386,13 +393,25 @@ func (d *Discovery) networkHandler() {
 		// through the (subjectively) new information on their own.
 		case syncReq := <-d.syncRequests:
 			nodePub := syncReq.node.SerializeCompressed()
-			log.Infof("Synchronizing channel graph with %x", nodePub)
-
 			if err := d.synchronize(syncReq); err != nil {
 				log.Errorf("unable to sync graph state with %x: %v",
 					nodePub, err)
 			}
 
+		//...
+		case <-reprocessTimer.C:
+			copyAnnouncementsToReprocess := d.announcementsToReprocess[:]
```

Comment:
I will rename it, but originally I meant that this is copy of the reference, rather than copy of the slice, I have done this in order to be able to `nil` slice so that it might be repopulated during re-processing.